pasion_preciouskaye/index.html

```diff
@@ -0,0 +1,186 @@
+          visitors to navigate. - Mikaella
+        </li>
+      </ul>
+    </div>
+  </div>
+    
+  <div class="goals">
+    <h2>My Goals in Life</h2>
+    <ul>
+      <li>Finish My Study</li>
+      <li>Get my Dream Job</li>
+      <li>Traveling with My Love ones</li>
+      <li>Buy our Own House</li>
+      <li>Make My Mother</li>
+    </ul>
+  </div>
+
+  <div class="yourcomment">
+    <h2> Your Comment About My Goals</h2>
+    <ul>
+      <li>Hope you'll achieve it all - Sofia</li>
+      <li>Great Job! - Eliza</li>
+      <li>You've got this! :D - Mikaella</li>
+    </ul>
+  </div>
+   
+    <div class="images">
+      <h2>Gallery</h2>
+      <div class="image-item">
+      <img
+        src="https://w0.peakpx.com/wallpaper/430/970/HD-wallpaper-doraemon-art-cute.jpg"
+      />
+      <img
+        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQyAv_4V8XzSLF2lTUeMpXS7p0nH-8QomfB5zzxh9INxw&s"
+      />
+      <img
+        src="https://i.pinimg.com/originals/05/f6/05/05f605183b35d08386e122bf5a767d63.jpg"
+      />
+      <img
+        src="https://yt3.googleusercontent.com/HskYwAGlO-VcPpnUNJcfTKBbFrj2LNrRbrm1Ht5cuHhc621f2Tgzf-deKf3uL0qMY8_KRApMoZg=s900-c-k-c0x00ffffff-no-rj"
+      />
+      <img
+        src="https://i.pinimg.com/236x/b7/91/a9/b791a92b8fc7819738019516d168ed25.jpg"
+      />
+      <img
+      src="https://i.pinimg.com/736x/69/5f/2f/695f2f95029f73e501cf28263b5aa492.jpg"
+    />
+     </div>
+    </div>
+
+    <div class="thoughts">
+      <h2>Face-toFace or Online Class</h2>
+        <p> better to be face to face class than online, because there 
+          are many advantage you'll be able to communicate with your 
+          instructors and classmates, and you'll be able to communicate with 
+          your classmates you will learn more and more because you can
+          see how your instructor is doing and you can hear your own instructor
+          speak forward. While online classes have many disadvantages, but 
+          there are also some advantages. The disavantage students are often
+          lazy and when the camera is off they don't listen, sleep, or do 
+          anything else it is not a matter of the subject, because the 
+          interaction between the students and the instructor is less 
+          understandable and audible.
+        </p>
+    </div>
+
+    <div class="mcomment">
+      <form class="mycomment" id="comment" oninput="mycomment()">
+        <h2>Comment</h2>
+        <div>
+          <label for="name">Name: </label>
+          <input type="text" id="name">
+        </div>
+    
+        <div>
+          <p>Comment: </p>
+          <textarea name="ycomment" id="ycomment"
+          cols="35" rows="10"></textarea>
+        </div>
+    
+        <div>
+          <button type="submit" id="button_comment" disabled>Submit</button>
+        </div>
+      </form>
+      
+      <script src="./script.js"></script>
```

Comment:
https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L141

It's quite unusual to see this here. Please put this line of code before the closing tag for body.